docs/usage/getting-started/running.md

```diff
@@ -1,70 +1,94 @@
+You can run this package in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+
+Renovate's npm package does not bundle any third-party tools or languages like Ruby, Python, Composer, Bundler, Poetry, etc.
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
 
 The `renovate` npm package is compatible with all of Renovate's supported platforms.
 
-Renovate requires Node.js `>=14.15.0` and Git `>=2.33.0`.
+Renovate requires Node.js `>=18.12.0` and Git `>=2.33.0`.
+
+#### Docker images
+
+Renovate is also distributed as Docker images on Docker Hub (`renovate/renovate`) and GitHub container registry (`ghcr.io/renovatebot/renovate`).
+These Docker images work on all the hosting platforms that Renovate supports.
+
+You must run the Docker images in a `linux/amd64` container.
+We're working on `linux/arm64` container support.
+You can't run the Docker images on a Windows or macOS container.
+
+In general, you can run Renovate on Windows with global mode.
+
+There are two flavors:
 
-#### Docker image
+- The `-slim` image
+- The default "full" image, this is the image you'll get if you use the `latest` tag
 
-The `renovate` npm package is also distributed via pre-built Node.js images on Docker Hub (`renovate/renovate`).
+##### The slim image
 
-The `slim` image contains only Node.js so works if either:
+The `-slim` image only comes with the Node.js environment.
+By default, the image installs the required tools when needed.
 
-- You do not require any additional package managers, or
-- You map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when necessary
+If you want, you can map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when needed.
 
-The "full" image (which `latest` defaults to) has every package manager which Renovate supports already preinstalled.
-This approach works best for many, but does have the following downsides:
+##### The default image
 
-- It only has _one_ version of each language/manager - usually the latest
-- It's several gigabytes in size
+The default image comes with every package manager that Renovate supports.
+This makes the default image the best choice for most users.
 
-The `renovate/renovate` Docker images are compatible with all of Renovate's supported platforms.
+You'll need to change to global mode, or Renovate still installs the latest version of the tools or the requested version from the repository.
+The pre-installed tool are sometimes outdated if the user doesn't update the Docker image regularly.
+
+We also support a persistent cache for downloaded tools, so that we only need to unpack on second+ run.
```

Comment:
should we link to some extended description on containerbase?

Reply:
I can't find more docs on `containerbase` in our Renovate docs search. Do we have more docs somewhere else?

Reply:
https://docs.renovatebot.com/self-hosted-configuration/#containerbasedir